CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to `src-cli` are documented in this file.
 
 ### Added
 
+- `SRC_HEADER_NAME=value` is now supported for authenticating `src` with custom auth proxies. See [auth proxy configuration docs](AUTH_PROXY.md) for more information.
```

Comment:
Same here, I'd use a concrete header instead of `_NAME`. That makes more sense in the context of "auth proxy configuration"